tests/test_timeline_monitor.py

```
@@ -0,0 +1,180 @@
+        # Verify UI display was called
+        mock_ui_instance.display_timeline.assert_called_once()
+
+    @patch('ccmonitor.timeline_monitor.load_sessions_in_timerange')
+    def test_run_with_project_filter(self, mock_load_sessions):
+        """Test run with project filter."""
+        mock_load_sessions.return_value = []
+        
+        monitor = TimelineMonitor(days=3, project="specific_project")
+        
+        with patch.object(monitor.console, 'print') as mock_print:
+            with patch.object(monitor.console, 'clear') as mock_clear:
```

Flake8 (7.2.0)

[error] local variable 'mock_clear' is assigned to but never used

(F841)